src/code/bank2.asm

```diff
@@ -6296,15 +6296,24 @@ clearIncrementAndReturn::
     ld   [wIgnoreLinkCollisionsCountdown], a      ; $6E0F: $EA $3E $C1
     jp   CheckForLedgeJumpAndReturn               ; $6E12: $C3 $45 $6E
 
-Data_002_6E15::
+; The following 16 bytes are coordinates of points on Link's sprite
+; used for collision detection with tiles. The coordinates are
+; measured in pixels from the top left corner of Link's 16 by 16 pixel
+; sprite. The first 8 bytes are x coordinates, the other 8 bytes are y
+; coordinates. The first two points are for case when Link is moving up,
+; the next two for moving right, the next two for moving down and the
+; last two for moving left.
+LinkCollisionPointsX::
     db   $06, $09, $0B, $0B, $06, $09, $04, $04
```

Comment:
What we sometimes do in the project is to indicate the different sub-sections of the data using labels:

```
.up    db $06, $09
.right db $0B, $0B
.down  db $06, $09
.left  db $04, $04
```

We even can display them in decimal rather than hex, it makes more sense (as we are counting pixels) 👍 

```suggestion
.up    db 06, 09
.right db 11, 11
.down  db 06, 09
.left  db 04, 04
```